### DISCUSSION TITLE
Virtualization explained

### DISCUSSION THREAD

**Discussion by Bristerboy911** [gregmac/Genmon-ESP32-Serial-Bridge#2] [Q&A] [answered]
Hi Greg! Very interested in your implementation.
Just wanted to get clarification...you're using ProxMox as hypervisor, then Alpine as a VM, and then have genmon running in LXC from inside Alpine, correct?

I'll be using VMWare so my implementation will be similar but not exact to yours.

Thanks!

**Comment by gregmac** [comment, marked as answer]
Proxmox runs the LXC container (see https://pve.proxmox.com/wiki/Linux_Container), and Alpine is the distro inside the container.

Note that none of this is strictly necessary; so long as genmon is running _somewhere_ and has network connectivity to the esp32, it will work.

**Comment by imweinstein** [comment]
Hi All,

I've tried to install Genmon in a Proxmox LXC container using Alpine.  I also tried to install Genmon on Ubuntu running in a VM on Proxmon.  Neither method is working.

The installs seem proceed normally (with some errors flashing by quickly on the screen), but nothing happens at that point.  When I go to the IP address of the VM (or Container) at http://ipaddress:8000, nothing comes up.

I've attached some screenshots showing some of the errors I'm seeing.  There are many more justl like this.  As you can see I followed the instructions to the letter.

Gregmac's instructions say to "Start Genmon" and update the config.  How does one start Genmon?

Are there more detailed instructions somewhere?  All of the instructions assume a VERY strong Linux background, which I don't have.  And the Youtube videos out there spend 95% of their time on the Pi / ESP hardware - not on installing Genmon on Linux.

Thank you!

Ira

![2023-01-08 14_50_56-pve - Proxmox Virtual Environment](https://user-images.githubusercontent.com/77629779/211216560-6604c2f2-fc12-4b66-a859-6df968f72149.jpg)

![2023-01-08 14_52_10-pve - Proxmox Virtual Environment](https://user-images.githubusercontent.com/77629779/211216559-c7cad7c5-0a45-45df-88a7-7c9d6ae0ddbf.jpg)

![2023-01-08 14_53_11-pve - Proxmox Virtual Environment](https://user-images.githubusercontent.com/77629779/211216558-c99998dd-42d9-44c4-b9e2-6bfdc3ec0c37.jpg)
